news/_posts/2020-01-07-critical-security-vulnerability-in-prestashop-modules.md

```diff
@@ -93,3 +95,57 @@ Modules and themes by other vendors may be vulnerable as well. Expect updates to
 
 A very special thank you to Francis Ladeuil who first reported this issue.
 
+## FAQ
+
+You can find answers to some frequent questions asked these past days.
+
+<details style="padding: .5em .5em 0;">
+  <summary style="margin: -.5em -.5em 0; padding: .5em;">**Which version is affected?**</summary>
+  All versions.
+  <br>The security vulnerabiltiy impacts modules, so it can affect any version using the vulnerable modules.
+  <br><br>A shop powered by PS 1.6, for example, can also be vulnerable if you install a vulnerable module which contains a phpunit directory.
+</details>
+
+<details style="padding: .5em .5em 0;">
+  <summary style="margin: -.5em -.5em 0; padding: .5em;">**Do I have to delete the Symfony related folder /vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/?**</summary>
+  No.
+  <br>Only the folders &lt;prestashop_directory>/vendor  and &lt;prestashop_directory>/modules/&lt;module_name>/vendor can  contain the vulnerable folder phpunit.
+  The folder /vendor/symfony/symfony/src/Symfony/Bridge/PhpUnit/ is safe **do not delete it**.
+</details>
+
+<details style="padding: .5em .5em 0;">
+  <summary style="margin: -.5em -.5em 0; padding: .5em;">**Do I need to upgrade my 1.6 to 1.7 to be safe?**</summary>
+  No.
+  <br>To secure you store follow the instructions.
+  <br><br>Though, we always recommend using the latest version as we regularly fix bugs and improve the security. <a href="https://www.prestashop.com/en/blog/maintenance-extension-prestashop-1-6">PrestaShop 1.6 is not maintained since July 2019</a>.
```

Comment:
I'm not sure though is used at the start of a sentence:
```suggestion
  <br><br>However, we always recommend using the latest version as each new version includes bug fixes and security improvements. <a href="https://www.prestashop.com/en/blog/maintenance-extension-prestashop-1-6">PrestaShop 1.6 is no longer officially maintained since July 2019</a>.
```